src/nunit.analyzers/UseCollectionConstraint/UseCollectionConstraintCodeFix.cs

```diff
@@ -37,18 +37,19 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var node = root.FindNode(context.Span);
 
-            if (!(node is ArgumentSyntax actualArgument) ||
-                !(actualArgument.Expression is MemberAccessExpressionSyntax actualExpression) ||
-                !(actualArgument.Parent is ArgumentListSyntax argumentList) ||
-                !(argumentList.Arguments.Count > 1 && argumentList.Arguments[1] is ArgumentSyntax constraintArgument))
+            if (node is not ArgumentSyntax actualArgument ||
+                actualArgument.Expression is not MemberAccessExpressionSyntax actualExpression ||
+                actualArgument.Parent is not ArgumentListSyntax argumentList ||
+                argumentList.Arguments.Count == 1 ||
```

Comment:
Shouldn't this be `argumentList.Arguments.Count <= 1 ||` ?

Before this change we would return if `argumentList.Arguments.Count == 0` now I think we would fail as we are accessing index 1